### DISCUSSION TITLE
Disable charge macbook manualy

### DISCUSSION THREAD

**Discussion by njfamirm** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1300] [Q&A]
Hi, I'm connect to my external display using USB-C port, I want to manually handle charging, but discharge option have limit when the battery percent fewer than limit, how can fix it?

just able to use the battery when I'm say

Thanks

**Comment by MatthiasKerbl** [comment]
Hi @njfamirm,

I may not understand your question correctly. Could you explain it again in more detail?
If you want to use the Discharge feature, make sure to set the Charge Limit to the desired level, below the current battery percentage, and click Discharge afterward.

Best,
Matthias

**Comment by njfamirm** [comment]
Thanks @MatthiasKerbl 

I'm just want to able use discharge mode without any limit, it is possible?

**Reply by MatthiasKerbl** [reply]
Discharging without a limit would result in the MacBook discharging completely and turning off. Therefore, a lower limit is required.

Best,
Matthias

**Reply by njfamirm** [reply]
so not possible, ok thnaks
